code/modules/client/preference_setup/loadout/lists/augmentations.dm

```diff
@@ -1,4 +1,33 @@
 /datum/gear/augmentation
 	sort_category = "Augmentations"
 	category = /datum/gear/augmentation
-	implant_into = BP_CHEST
+	flags = GEAR_NO_EQUIP | GEAR_NO_FINGERPRINTS
+	custom_setup_proc = /obj/item/proc/attempt_implant
+	custom_setup_proc_arguments = list(BP_CHEST)
+
+
+/obj/item/proc/attempt_implant(mob/user, target_zone)
+	to_chat(user, SPAN_DANGER("Was unable to implement you with \the [src]"))
+	qdel(src)
+
+/obj/item/implant/attempt_implant(mob/user, target_zone)
+	if(can_implant(user, user, target_zone) && implant_in_mob(user, user, target_zone))
+		var/obj/item/organ/organ = user.get_organ(target_zone)
+		to_chat(user, SPAN_NOTICE("You have \a [src] implemented in your [organ.name]"))
+		return
+	..()
+
+/obj/item/organ/internal/augment/attempt_implant(mob/living/carbon/human/user, target_zone)
```

Comment:
This proc is probably partially copy-paste from surgery code and is thus questionable